src/controlllers/AgentSignup.js

```diff
@@ -0,0 +1,44 @@
+import bcrypt from 'bcrypt';
+import jwt from 'jsonwebtoken';
+import Model from '../models/model';
+
+const agentModel = new Model('agents');
+
+export const createAgent = async (req, res) => {
+  const saltRounds = 5;
+  const {
+    firstName, lastName, email, password, phoneNo
+  } = req.body;
+  const hashedPassword = await bcrypt.hash(password, saltRounds);
+  const columns = 'first_name, last_name, email, password, phone_no';
+  const values = `'${firstName}', '${lastName}', '${email}', '${hashedPassword}', '${phoneNo}'`;
+  try {
+    const validEmail = await agentModel.select('*', ` WHERE  email = '${email}' `);
+    if (validEmail.rows.length > 0) {
+      return res.status(409).json({ messages: 'Email already Exist' });
+    }
+    const data = await agentModel.insertWithReturn(columns, values);
+    const token = jwt.sign({ id: data.rows.id }, process.env.TOKEN_KEY, {
+      expiresIn: '1m',
+    });
+
+    res.header('user-created', token).status(200).send({
+      token, firstName, lastName, email
+    });
+  } catch (err) {
+    res.status(400).json({ messages: err.stack });
+  }
+};
+
+export const logAgent = async (req, res) => {
```

Comment:
Give this function a more descriptive name like `loginAgent` instead of `logAgent`